src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -36,4 +36,65 @@
     </xsd:complexType>
     <xsd:element name="ImportCertificateResponseMessage" type="tns:ImportCertificateResponseMessageType"/>
     
+    
+    <xsd:complexType name="ListTrustStoresRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListTrustStoresRequestMessage" type="tns:ListTrustStoresRequestMessageType" />
+    
+    <xsd:complexType name="ListTrustStoresResponseMessageType">
+         <xsd:sequence>
+            <xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream" minOccurs="0" maxOccurs="unbounded"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListTrustStoresResponseMessage" type="tns:ListTrustStoresResponseMessageType"/>
+    
+    <xsd:complexType name="EditCertificateRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="editCertRequest" type="tns:EditCertificateRequestType" minOccurs="1" maxOccurs="1" />
+            <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="EditCertificateRequestMessage" type="tns:EditCertificateRequestMessageType" />
+    
+    <xsd:complexType name="EditCertificateRequestType">
+        <xsd:sequence>
+            <xsd:element name="alias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="EditCertificateRequest" type="tns:EditCertificateRequestType" />
+        
+    <xsd:complexType name="EditCertificateResponseMessageType">
+        <xsd:sequence>
+            <xsd:element name="status" type="xsd:boolean"/>
+			<xsd:element name="message" type="xsd:string"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="EditCertificateResponseMessage" type="tns:EditCertificateResponseMessageType"/>
+    
+	 <xsd:complexType name="DeleteCertificateRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="deleteCertRequest" type="tns:DeleteCertificateRequestType" minOccurs="1" maxOccurs="1" />
+            <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="DeleteCertificateRequestMessage" type="tns:DeleteCertificateRequestMessageType" />
+    
+    <xsd:complexType name="DeleteCertificateRequestType">
+        <xsd:sequence>
+            <xsd:element name="alias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="DeleteCertificateRequest" type="tns:DeleteCertificateRequestType" />
+        
+    <xsd:complexType name="DeleteCertificateResponseMessageType">
+        <xsd:sequence>
```

Comment:
I think we need to have a "SimpleCertificateResponse"  since we repeat the same status,message object for 3 different services.  Would replace the response type for delete, import, and edit alias.  Same for refresh.